diktat-rules/src/test/kotlin/com/saveourtool/diktat/ruleset/chapter1/IdentifierNamingWarnTest.kt

```diff
@@ -629,6 +636,33 @@ class IdentifierNamingWarnTest : LintTestBase(::IdentifierNaming) {
         lintMethod(code)
     }
 
+    @Test
+    @Tag(WarningNames.VARIABLE_NAME_INCORRECT_FORMAT)
+    fun `should not trigger on backing field`() {
```

Comment:
please add more positive examples, when it is triggered on the backing field

Reply:
Done